packages/utils/src/getAccountsFromLocalStorage.ts

```diff
@@ -0,0 +1,22 @@
+import { AuthorizeResponse } from '@deriv/api-types';
```

Comment:
```suggestion
import type { AuthorizeResponse } from '@deriv/api-types';
```
@adrienne-deriv Need to add `@deriv/api-types` to `devDependencies`.